docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -12,7 +12,7 @@ By default, the command will print the content of the unit files to stdout.
 
 Generating unit files for a pod requires the pod to be created with an infra container (see `--infra=true`).  An infra container runs across the entire lifespan of a pod and is hence required for systemd to manage the life cycle of the pod's main unit.
 
-_Note: If you use this command with the remote client, including Mac and Windows (excluding WSL2) machines, you would still have to place the generated units on the remote system.  Moreover, please make sure that the XDG_RUNTIME_DIR environment variable is set.  If unset, you may set it via `export XDG_RUNTIME_DIR=/run/user/$(id -u)`._
+_Note: If you use this command with the remote client, including Mac and Windows (excluding WSL2) machines, you would still have to place the generated units on the remote system.  Moreover, please make sure that the `XDG_RUNTIME_DIR` environment variable is set.  If unset, you may set it via `export XDG_RUNTIME_DIR=/run/user/$(id -u)`._
```

Comment:
Minor nit: Trailing whitespace at end of this line. Was there before, apparently, kind of surprised CI didn't catch it.

Reply:
That's an underscore, matching the opening underscore in `Note`.

(My editor doesn't let me save files with trailing whitespace)

Reply:
Ah, OK